sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/EditContactSteps.java

```diff
@@ -98,6 +100,133 @@ public EditContactSteps(WebDriverHelpers webDriverHelpers) {
           softly.assertAll();
         });
 
+    When(
+        "I check the edited data is correctly displayed on Edit Contact page after editing",
+        () -> {
+          aContact = collectContactDataAfterEdit();
+
+          SoftAssertions softly = new SoftAssertions();
```

Comment:
Please inject the object instead of creation.

Reply:
reworked.